src/payment/strategies/adyenv2/adyenv2.ts

```diff
@@ -779,5 +779,5 @@ export type AdyenComponentState = (
 
 export default function isCardState(param: any): param is CardState {
     return param && typeof param.data.paymentMethod.encryptedSecurityCode === 'string' ||
-        typeof param.data.paymentMethod.encryptedExpiryMonth === 'string';
+        typeof param.data.paymentMethod.encryptedExpiryMonth === 'string' || param.data.paymentMethod.type === 'ideal';
```

Comment:
I think this doesn't fit here very well, you should create another guard validating if it is `AccountState` or something like that. Of course, you also need to define the new interface, because `CardState` is not an appropriate one.